client/src/components/searchResults/index.js

```diff
@@ -1,31 +1,94 @@
 import React from 'react'
-import { FlatButton } from 'material-ui'
-import DisplayTable from './displayTable'
+import { FlatButton, Table, TableBody } from 'material-ui'
+import PropTypes from 'prop-types'
+import './style.css'
+import { renderUser, renderOrganization, renderProgram } from './rows'
 
-/* dummies */
-import dummyList from './dummies'
+let displaySearchKey
 
-import './style.css'
+const displaySearchKeyMessage = searchKey => {
+  if (displaySearchKey !== '') {
+    return <p>Results for {`"${searchKey}"`}</p>
+  }
+  return <p>Enter a search term</p>
+}
```

Comment:
could we do the check on `searchKey` instead? That way we don't have a dangling `let displaySearchKey` variable in this file.